### ISSUE TITLE
Jenkins Pipeline 'tool' step not returning 'home'

### ISSUE THREAD

**Issue by alvin-huang** [dpires/jenkins-terraform-plugin#5]
I am having an issue with the Jenkins Pipeline tool step being able to return the homedir of where terraform is installed. In examples, I see:

```
def mvnHome = tool 'M3'
sh "${mvnHome}/bin/mvn -B verify"
```

When I try this with my installed maven plugin, it returns mvnHome correctly. However, when I try this with my terraform install (automatic install from bintray) I am seeing nothing returned as the tfHome for something like:

```
def tfHome = tool 'Terraform'
```

Furthermore, I see that in the XML for the plugin:

```
    <org.jenkinsci.plugins.terraform.TerraformInstallation>
      <name>Terraform</name>
      <home></home> <-------NOTICE 
      <properties>
        <hudson.tools.InstallSourceProperty>
          <installers>
            <org.jenkinsci.plugins.terraform.TerraformInstaller>
              <id>0.7.1-linux-amd64</id>
            </org.jenkinsci.plugins.terraform.TerraformInstaller>
          </installers>
        </hudson.tools.InstallSourceProperty>
      </properties>
    </org.jenkinsci.plugins.terraform.TerraformInstallation>
```

I think the getHome() is not returning anything but I am not 100% sure. Any ideas on this?

**Comment by dpires** [comment]
Interesting, I'm not sure why home isn't being set. What version of Jenkins are you using? I'll take a look into this.

**Comment by alvin-huang** [comment]
I'm on 2.10

**Comment by dpires** [comment]
Can you manually edit the xml config and set the home value to see if the Pipeline picks it up?

If that is the case It is probably an issue with how I am using the Installer or a the Installer API has changed.

**Comment by alvin-huang** [comment]
Yup, looks like it works when I set it in the XML, restart Jenkins, and grab it using the tool step, so that seems to be where it is breaking.

**Comment by dpires** [comment]
Okay, thanks. I'll look into what needs changing/updating to get that home setting set.

**Comment by dpires** [comment]
I have implemented a fix for this in 1.0.10.
